crates/net/peers/Cargo.toml

```
@@ -25,13 +25,15 @@ secp256k1 = { workspace = true, optional = true }
 serde_with.workspace = true
 thiserror.workspace = true
 url.workspace = true
-tokio = { workspace = true, features = ["full"] }
+tokio = { workspace = true, optional = true }
 
 [dev-dependencies]
 alloy-primitives = { workspace = true, features = ["rand"] }
 rand.workspace = true
 secp256k1 = { workspace = true, features = ["rand"] }
 serde_json.workspace = true
+tokio = { workspace = true, features = ["net", "macros", "rt"] }
 
 [features]
 secp256k1 = ["dep:secp256k1", "enr/secp256k1"]
+net = ["dep:tokio", "tokio?/net"]
```

```suggestion
net = ["dep:tokio", "tokio*/net"]
```
isn't it Tokio and any other libs like `tokio-stream` we want to match on?

---

ah optional dep in features section!